java/client/src/main/java/glide/connectors/handlers/MessageHandler.java

```diff
@@ -59,11 +61,13 @@ public void handle(Response response) {
                         "Transport disconnected, messages might be lost");
                 break;
             case PMessage:
-                handle(new PubSubMessage((String) values[2], (String) values[1], (String) values[0]));
+                handle(
+                        new PubSubMessage(
+                                gs((String) values[2]), gs((String) values[1]), gs((String) values[0])));
```

Comment:
```suggestion
                                (GlideString) values[2], (GlideString) values[1], (GlideString) values[0]));
```
You probably need to use another value resolver (see line 43)